src/main/java/com/pollapp/controller/PollController.java

```diff
@@ -36,18 +37,26 @@
     @Autowired
     private PollProcess pollProcess;
 
+    @GetMapping("/all")
+    public List<PollResponse> getAllPolls() {
```

Comment:
@GetMapping("") is more appropriate for rest api

Reply:
I can change it to "/polls" but some distinct address is needed for it to cooperate efficiently with categories. The problem is that the address will then be "polls/polls".

Reply:
Whole controller has already /polls mapping. Under this endpoint we want to get all polls so we can just leave empty bracket. For getting all polls by category url should be like this /polls/categories/{categoryId}

Reply:
OK, I sorted it out. It now works well with ("").